tools/run_tests/build_python.sh

```diff
@@ -38,5 +38,5 @@ rm -rf python2.7_virtual_environment
 virtualenv -p /usr/bin/python2.7 python2.7_virtual_environment
 source python2.7_virtual_environment/bin/activate
 pip install enum34==1.0.4 futures==2.2.0 protobuf==3.0.0-alpha-1
-CFLAGS=-I$root/include LDFLAGS=-L$root/libs/$CONFIG pip install src/python/src
+CFLAGS="-I$root/include" LDFLAGS="-L$root/libs/$CONFIG" pip install src/python/src
```

Comment:
The change on this line smells entirely independent of the whole rest of this commit. May this be made in a separate pull request?